### PR TITLE
Update documents layer render visibility

### PR DESCRIPTION
**Pull request by junqiu-lei**
Signed-off-by: Junqiu Lei <junqiu@amazon.com>

### Description
This PR fixes the issue documents layer visibility couldn't be rendered by maps saved object visibility property.

### Demo

https://user-images.githubusercontent.com/90288540/206567005-7c28029f-c9bf-48f7-bcb2-084eead4d805.mov

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by VijayanB** [comment]
Can you add demo?

**Comment by junqiu-lei** [comment]
> Can you add demo?

Updated on the description

File: maps_dashboards/public/model/documentLayerFunctions.ts [hunk 1]
```diff
@@ -112,14 +113,16 @@ const addNewLayer = (
...
   };
 
   const addGeoShapeLayer = (source: any) => {
     source.features.map((feature: any) => {
       const mbType = GeoJSONMaplibreMap.get(feature.geometry.type);
       if (mbType === 'circle') {
-        maplibreRef.current?.addLayer({
-          id: layerConfig.id + '-' + feature.properties.title,
+        const circleLayerId = layerConfig.id + feature.properties.title;
+        maplireInstance?.addLayer({
```
Comment (VijayanB) [inline maps_dashboards/public/model/documentLayerFunctions.ts:124]: maplbireInstance

Comment (junqiu-lei) [reply maps_dashboards/public/model/documentLayerFunctions.ts]: updated

**Review by VijayanB — APPROVED** [review]

**Review by heemin32 — APPROVED** [review]

**Pull request merged**